HomeTask/HomeTask/TaskList/TaskListView.swift

```diff
@@ -27,10 +27,16 @@ struct TaskListView: View {
                     }
                     
                 case .content:
-                    List(viewModel.tasks) { task in
-                        Text(task.title)
+                    List {
+                        ForEach(viewModel.tasks) { task in
+                            Text(task.name ?? "no name")
```

Comment:
Idealmente, os modelos que são usados pela view já estão prontos para uso, a view não deve tratá-los de nenhuma forma (lembrando, a view é o mais burra possível), como verificar se uma propriedade é vazia e, caso seja, usar algum outro valor padrão.
Então, isso tudo seria lidado pela view model. O modelo principal (que aqui você chamou de `TaskItem`, não seria visível pela view, mas seria encapsulado pela view model em um modelo já tratado para exibição (poderíamos chamar de `CellItem`, por exemplo).
Então aqui seria algo do tipo:
```suggestion
                        ForEach(viewModel.taskCellItems) { cellitem in
                            Text(cellItem.title)
```

Reply:
botei um post-it do lado do monitor "View não pensam, elas mostram"